packages/fast-components-react-msft/src/hypertext/README.md

```diff
@@ -1,11 +1,8 @@
-## Best practices
-Use *hypertext* when you need to navigate the user to more information about the text that was selected. *Hypertexts* are best used for tertiary actions outside the main work flow.
+## Guidance
+The *Hypertext* component is an [anchor](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) element which accepts children. *Hypertexts* most commonly appear as inline elements within the context of a sentence or a paragraph.
```

Comment:
Same with all of these

Reply:
I would assume the whole file based on the previous comment :)

Reply:
resolved.